Hazel/Dtls/DtlsConnectionListener.cs

```diff
@@ -246,27 +248,7 @@ public void SetCertificate(X509Certificate2 certificate)
             this.certificatePrivateKey = privateKey;
 
             // Pre-fragment the certificate data
-            ByteSpan certificateData = Certificate.Encode(certificate);
```

Comment:
There's a big multiline comment ~line 929 referring to this SetCertificate message - might need to update that